docs/src/catalyst_applications/advanced_simulations.md

```diff
@@ -217,69 +226,67 @@ because the callback modifies our `ODEProblem` during the simulation, and this
 modification remains during the second simulation. An improved workflow to avoid
 this issue is:
 ```@example ex2
```

Comment:
Still think this will use the old (modified) p, so at least that must be added back in.